common/amundsen_common/models/search.py

```diff
@@ -28,6 +28,8 @@ class SearchRequest:
     page_index: Optional[int] = 0
     results_per_page: Optional[int] = 10
     filters: List[Filter] = []
+    highlight_matches: bool = False
```

Comment:
single "highlighting_options" field is better.

Reply:
so does that have a subfield like `highlight_matches`? I want to make sure we provide an option to just enable highlight with some default fields if the field is set to`true`, and then override that functionality if `false`.

Reply:
yes, has nested field like `enabled: true` and others.